src/exportable/ExportableMappingRule.ts

```diff
@@ -0,0 +1,15 @@
+import { fshrules } from 'fsh-sushi';
+import { ExportableRule } from '.';
+
+export class ExportableMappingRule extends fshrules.MappingRule implements ExportableRule {
+  constructor(path: string) {
+    super(path);
+  }
+
+  toFSH(): string {
+    const path = this.path ? `${this.path} ` : '';
+    const comment = this.comment ? ` "${this.comment}"` : '';
+    const language = this.language ? ` ${this.language.toString()}` : '';
+    return `* ${path}-> "${this.map}"${comment}${language}`;
```

Comment:
Now might be the time when we need to properly decide how to use `fshifyString`, since currently it is sometimes used on the early side (AssignmentRuleExtractor, getFSHValue by way of CaretValueRuleExtractor) and sometimes on the late side (metadataToFSH). What you've written is correct if strings are always fshified as early as possible. Please let me know what you think about this.

Reply:
Hmm that's a good point that I hadn't thought about. I don't think I have a super strong opinion beyond being consistent. If I had to choose, I guess ensuring that `fshifyString` gets called in the MappingRuleExtractor makes sense and follows the two other rule extractors you pointed out. That does seem to differ a bit from how metadataToFSH handles it, so if we wanted to go for consistency, we'd have to change those. Do you have a preference for which way we lean?